src/Components/LinkCreator/LinkCreator.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+
+import { checkUrlAvailable } from "../../utils/api.js";
+import { vocascanServer } from "../../utils/constants.js";
+
+import "./LinkCreator.scss";
+
+const LinkCreator = ({ path, setValid, electronFix = false, children }) => {
+  const selfHosted = useSelector((state) => state.login.selfHosted);
+  const serverAddress = useSelector((state) => state.login.serverAddress);
+  const appLanguage = useSelector((state) => state.setting.language);
+
+  const [url, setUrl] = useState("");
+
+  const { BASE_URL: baseURL, ENV: env } = window.VOCASCAN_CONFIG;
+
+  //initialize url with the needed url for the servers
```

Comment:
can you please add a space after such comments.
```suggestion
  // initialize url with the needed url for the servers
```